### ISSUE TITLE
Can't find variable: exports

### ISSUE THREAD

**Issue by seldimi** [ganlanyuan/tiny-slider#730]
After version 2.9.2, each time we use dist or src tiny-slider.js, there following error is on console:
Can't find variable: exports

CDN is 2.9.2 so I cannot compare. 
Any ideas?

**Comment by QuestionDevelopment** [comment]
Looks like exports are in a file called browserstack.conf.js.

You can try loading that first however I just added this to the top of the js file.  Not 100% sure the consequences to this so I don't recommended it.

```
var exports = {};
```

**Comment by seldimi** [comment]
I’m sorry I haven’t understood. Following the simple install steps with npm, without any other dependencies tns won’t be useable and we will have to put that initialization array before import?

https://github.com/ganlanyuan/tiny-slider#install

**Comment by ETNyx** [comment]
Try to use tiny-slider.js from dist/min directory

**Comment by tadywankenobi** [comment]
I had the same, it was because I was instantiating the slider carousel inside a jQuery document.ready. Try moving this outside of jQuery.

**Comment by seldimi** [comment]
I see.
But jQuery is widely spread (still) so there should be another workaround.
For the time being, I use webpack with dist folder

**Comment by bariscanyilmaz** [comment]
> Try to use tiny-slider.js from dist/min directory

this is solved my issue thanks

**Comment by marceloverdijk** [comment]
Yes using the tiny-slider.js from dist/min directory works, but ...
it's the minified version and sometime for debugging the non minified version is useful.
